### ISSUE TITLE
"group news" dead link displayed in user profile owner block

### ISSUE THREAD

**Issue by oseg** [iionly/news#1]
versions: elgg v1.12.4, news plugin v1.11.5
context: The issue is only reproduced with non-admin users
scenario: display user profile page
potential fix in function news_owner_block_menu in start.php file:

```
} else {
    if (!elgg_instanceof($params['entity'], 'user') && ($params['entity']->news_enable != "no") 
```

**Comment by iionly** [comment]
Thanks for reporting. It seems the me that I messed up the whole code of the news_owner_block_menu() function. At least it doesn't make sense to me now looking at it. The point is that I seem to have missed the fact that it isn't important if the user who is viewing the profile page / hover menu is admin or group owner but rather the user whose profile page / hover menu you are viewing would need to be an admin or group owner. Other users can't make news postings anyway and therefore wouldn't need to have a menu entry added in the first place. I'll try to take a look at it and fix it. Though I would like to finish the updating of another plugin first.

**Comment by iionly** [comment]
Fixed in version 1.11.6 / 2.0.1.
